api/v1/controllers/samples.js

```diff
@@ -31,6 +31,42 @@ const utils = require('./utils');
 const publisher = u.publisher;
 const kueSetup = require('../../../jobQueue/setup');
 const kue = kueSetup.kue;
+const getSamplesWildcardCacheInvalidation = require('../../../config')
+  .getSamplesWildcardCacheInvalidation;
+const redisCache = require('../../../cache/redisCache').client.cache;
+
+/**
+ * Function for finding sample from samplestore. If cache is on then
```

Comment:
No need to say "function for..." in the function-level comment. Should just be `Find sample from samplestore. If cache is on then cache the response as well'